src/main/kotlin/MinecraftConfigurable.kt

```diff
@@ -91,6 +92,16 @@ class MinecraftConfigurable : Configurable {
             }
         }
 
+        group(MCDevBundle("minecraft.setting.translation")) {
```

Comment:
Nitpick, same for the two other keys below

```suggestion
        group(MCDevBundle("minecraft.settings.translation")) {
```

Reply:
[b3bd4e7](https://github.com/minecraft-dev/MinecraftDev/pull/2292/commits/b3bd4e76f7057f9dab92acc49449ad39e3f450a4)